app/src/constants/i18n.ts

```diff
@@ -331,11 +331,17 @@ export const SurveyAnimalsI18N = {
   animalMortalityHelp:
     "Mortality Events describe an individual's death, including the suspected location, date, and cause of death. An individual can only have one Mortality Event.",
   animalMortalityAddBtn: 'Add Mortality',
-
+  animalCollectionUnitTitle: 'Ecological Units',
+  animalCollectionUnitTitle2: 'Ecological Unit',
+  animalCollectionUnitHelp:
+    'Ecological units are groups like population units, herds, and packs. Different species may different units and unit names.',
```

Comment:
Nit: first sentence, I would use "such as " instead of "like"; Second sentence contains a typo

Reply:
Fixed in merge conflict